etc/helm/pachyderm/values.yaml

```diff
@@ -503,6 +507,8 @@ postgresql:
     size: 10Gi
     labels:
       suite: pachyderm
+  primary:
+    nodeSelector: {}
```

Comment:
I'm not clear if this needs to be here, or should be here by convention, as this is just passing the value through to the subchart

Reply:
where'd "primary" come from lol?

Reply:
ah, maybe I'm doing it wrong. It seemed like the pod-related stuff in the postegresql chart was under `primary:`, maybe I don't understand how that works

Reply:
so my bad - you're totally right here. It's how the postgresql subchart distinguishes between config for the primary write PG replica vs the read replicas that can be deployed https://github.com/pachyderm/pachyderm/blob/master/etc/helm/pachyderm/charts/postgresql/templates/statefulset-readreplicas.yaml. Could you also add a similar readReplicas section?

Reply:
ah, nice catch, that naming makes more sense now